### ISSUE TITLE
option combined with value

### ISSUE THREAD

**Issue by BebeSparkelSparkel** [SamuelSchlesinger/commander-cli#25]
Some options are easier to use if the option is combined with the value like the `sed` option `-i[extension]`.

Other CLIs use the assignment operator like `HandBrakeCLI` with the option `-v, --verbose[=number]  Be verbose (optional argument: logging level)`

**Comment by SamuelSchlesinger** [comment]
Yeah, I guess the assignment operator could do the trick here. I feel similarly to this as I do to combined flags.

**Comment by BebeSparkelSparkel** [comment]
I don't want to limit it to an assignment operator. Just matching a prefix key to the value is what I would suggest.

`optJoined @"prefix"` would drop "prefix" from the text `"prefix1234"` and then try to unrender the remaining text `"1234"`

**Comment by SamuelSchlesinger** [comment]
Yeah, that's even better. PRs welcome, yet again!

**Comment by BebeSparkelSparkel** [comment]
I'll wait until you review the other PRs
